src/tests/test_utils/functional_test_utils/layer_tests_summary/conformance_helper_tools/conformance_vs_accuracy_comparator.py

```diff
@@ -66,11 +68,21 @@ def process_model_list(model_list_file_path: os.path):
     with open(model_list_file_path, "r") as model_list_file:
         in_models = model_list_file.readlines()
         prefix = os.path.commonprefix(in_models)
+        prefix += '(.*?)/'
         for line in in_models:
             models.add(path_to_model(line, prefix))
         model_list_file.close()
     return models
 
+def convert_status_to_conformance_style(accuracy_status: str):
```

Comment:
`convert_accuracy_res_to_bool` and `convert_conformance_to_bool`? to align different status?

Reply:
done